### PR TITLE
Underline under the icon on hover

### PR DESCRIPTION
**Pull request by Ramanpreet6262**
Fixed underline under the icon on hover.
ref #30

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -23,9 +23,9 @@ a {
   padding-top: 5px;
 }
 
-a:not(.github):hover {
+/* a:not(.github):hover {
```
Comment (AnshulMalik) [inline css/style.css]: if we don't need it we can just remove it.

File: css/style.css [hunk 2]
```diff
@@ -77,3 +76,7 @@ div.grid-item {
   margin-bottom: 20px;
   margin-top: 10px;
 }
+
+.language-name:hover{
+  border-bottom: 3px solid #1b6568;
```
Comment (AnshulMalik) [inline css/style.css:77]: Two points here.
1. I think we are missing the little transition effect `transition-duration: 100ms;` here
2. New line.

**Review by AnshulMalik — CHANGES_REQUESTED** [review]
Just a few nits.

File: css/style.css (hunk 1 above)
Comment (Ramanpreet6262) [reply css/style.css]: Ok I'll remove it

File: css/style.css (hunk 2 above)
Comment (Ramanpreet6262) [reply css/style.css:77]: Transition effect I'll add.
What's new line?

File: css/style.css [hunk 3]
```diff
@@ -77,3 +76,7 @@ div.grid-item {
...
+}
```
Comment (AnshulMalik) [inline css/style.css]: The new line is just a lint thing, should have an empty line at the end of a file.

Comment (Ramanpreet6262) [reply css/style.css]: ok

**Review by AnshulMalik — APPROVED** [review]
Cool

**Pull request merged**